cider-dap/src/adapter.rs

```diff
@@ -199,26 +218,36 @@ impl MyAdapter {
             Some(p) => {
                 let out: Vec<Variable> = p
                     .iter()
-                    .map(|x| Variable {
-                        name: String::from(x),
-                        value: String::from("1"),
-                        type_field: None,
-                        presentation_hint: None,
-                        evaluate_name: None,
-                        variables_reference: 0,
-                        named_variables: None,
-                        indexed_variables: None,
-                        memory_reference: None,
+                    .map(|(nam, val)| {
+                        let valu = match val.as_option() {
```

Comment:
rather than a match you should be able to do something like
`val.as_option().map(|x| x.val().to_u64().unwrap()).unwrap_or_default()`